indico/modules/events/roles/blueprint.py

```diff
@@ -23,3 +24,6 @@
 
 _bp.add_url_rule('/<int:role_id>/members', 'add_members', RHAddEventRoleMembers, methods=('POST',))
 _bp.add_url_rule('/<int:role_id>/members/<int:user_id>', 'remove_member', RHRemoveEventRoleMember, methods=('DELETE',))
+
+_bp.add_url_rule('/<int:role_id>/members/import', 'add_members_import_csv', RHAddEventRoleMembersImportCSV,
```

Comment:
The same as before. Do you really need that `Add` part in tthe name of the `RH`?

Reply:
changed it.